src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+public enum Winning {
+	MATCH6(6, 2000000000),
+	MATCH5(5, 1500000),
+	MATCH4(4, 50000),
+	MATCH3(3, 5000),
+	NAN(0, 0);
+
+	private final int match;
+	private final int money;
+
+	Winning(int match, int money) {
+		this.match = match;
+		this.money = money;
+	}
+
+	public static Winning from(int match) {
+		if (match == 6) {
+			return Winning.MATCH6;
+		}
+		if (match == 5) {
+			return Winning.MATCH5;
+		}
+		if (match == 4) {
+			return Winning.MATCH4;
+		}
+		if (match == 3) {
+			return Winning.MATCH3;
+		}
+		return Winning.NAN;
```

Comment:
```suggestion
	return Arrays.stream(values())
	        .filter(rank -> rank.match == match)
	        .findFirst()
	        .orElse(NAN);
```
이렇게 수정해볼 수도 있겠네요 혹시라도 또다른 match count가 생겨나더라도 if문이 추가적으로 늘어나지 않는 등 조금 더 변경에 용이하고 확장성 있는 코드가 될 것 같아요 :)

아 그리구 현재 코드 컨벤션이 일반적이지 않으신거 같아요!
tab이나 indent size가 8로 되어있으신거같은데 4로 변경해주시면 감사하겠습니다 🙏

Reply:
저도 이 부분을 어떻게 해야할 지 고민했었는데
조언 주셔서 감사합니다!🙇🏻‍♂️

그리고 코드 컨벤션이 tab으로 되어있었더라구요😭
4스페이스로 변경하겠습니다!